docs/develop/setup.md

```diff
@@ -1,74 +1,183 @@
 # Setting up your development environment
 
-There are two ways to set up your development environment. You can view the [pros and
-cons of each method](#pros-and-cons-of-each-setup-method) at the bottom.
+There are two ways to set up your development environment for AAQ:
 
-## Set up using Docker Compose Watch
+1. [Docker Compose Watch](#set-up-using-docker-compose-watch)
+2. [Manual](#set-up-manually)
+
+You can view the [pros and cons of each method](#pros-and-cons-of-each-setup-method) at
+the bottom.
+
+Before you get started, please fork the
+[project repository](https://github.com/IDinsight/aaq-core) by clicking on the
+"Fork" button and then clone the repo using:
+
+    git clone git@github.com:<your GitHub handle>/aaq-core.git
+
+---
+**Note**
+
+For questions related to setup, please contact:
+
+- [Tony Zhao](mailto:tony.zhao@idinsight.org?Subject=AAQ%20Local%20Setup%20Help)
```

Comment:
🥺  thank you! We actually have an aaq email `aaq@idinsight.org` (which right now only forwards to Tanmay and Sid, but we can get on it too). Maybe we can create aaq-support@idinsight.org? we can discuss.

Reply:
Let's use aaq@idisnight.org as @Tanmay-97 suggested!